e2e/perf/post-pr-comment.py

```
@@ -0,0 +1,600 @@
+                old_content = extract_current_run_content(existing_comment['body'])
+
+                # Timestamp from when the comment was posted (UTC)
+                old_timestamp = _format_previous_run_timestamp(datetime.now(timezone.utc))
+                if 'created_at' in existing_comment:
+                    try:
+                        created_at = datetime.fromisoformat(
+                            existing_comment['created_at'].replace('Z', '+00:00')
+                        )
+                        old_timestamp = _format_previous_run_timestamp(created_at)
+                    except Exception:
+                        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)